lib/improver/tests/test_ensemble_copula_coupling_GeneratePercentilesFromProbabilities.py

```diff
@@ -461,67 +393,18 @@ def test_check_data_spot_forecasts(self):
         self.assertArrayAlmostEqual(result.data, data)
 
 
-class Test__get_bounds_of_distribution(IrisTest):
```

Comment:
Moved to lib/improver/tests/test_ensemble_copula_coupling_EnsembleCopulaCouplingUtilities.py